protocol/communities/manager.go

```diff
@@ -4497,8 +4499,8 @@ func (m *Manager) AddCommunityToken(token *community_token.CommunityToken, clock
 			Type:              protobuf.CommunityTokenType_ERC721,
 			Symbol:            token.Symbol,
 			Name:              token.Name,
-			Amount:            "1",
```

Comment:
Should we still add the `Amount` for old apps to still show the right value?

Reply:
Very good catch! We should.